src/pages/certifications/javascript-algorithms-and-data-structures/object-oriented-programming/change-the-prototype-to-a-new-object/index.md

```diff
@@ -2,9 +2,87 @@
 title: Change the Prototype to a New Object
 ---
 ## Change the Prototype to a New Object
+Instead of adding each prototype property one by one with object.prototype.property. WE cna do this much easier by setting the prototype to a new object. That way all of the prototype properties are added at once.
```

Comment:
Hi, could you add code formatting for `object.prototype.property` with backticks? Also, theres a typo with "WE cna", please fix that.